packages/http-smoke-testing/src/RouterAdapter/SymfonyRouterAdapter.php

```diff
@@ -13,32 +18,84 @@ class SymfonyRouterAdapter implements RouterAdapterInterface
      */
     private $router;
 
+    private $annotationsReader;
+
     /**
      * @param \Symfony\Component\Routing\RouterInterface $router
      */
     public function __construct(RouterInterface $router)
     {
+        AnnotationRegistry::registerLoader('class_exists');
```

Comment:
This should not be called in constructor

Reply:
I would leave it here. This should be temporary solution until doctrine/annotations v2.0 then it should be possible to use autoloader.

Reply:
hmm ok